admin/tabs/general/configs/ConfigModel.js

```diff
@@ -68,6 +75,19 @@ export class ConfigModel {
         actionWarning: {
             del: 'Are you sure you want to delete? Deleting configs can break running apps!'
         },
+        toolbarActions: [
+            addAction,
+            editAction,
+            cloneAction,
+            deleteAction
+        ],
+        menuActions: [
+            addAction,
+            editAction,
+            cloneAction,
+            deleteAction
+        ],
```

Comment:
I don't have a better proposal at the moment, but it does seem unfortunate that we need to specifically list all these out (in a particular, hopefully standard order) to enable the clone action.  

I understand how having the API take a list of actions for these various sections of the grid makes sense - and how boolean flags would rapidly spin out of control.  

Mentioning in case anyone had any ideas.  Happy to proceed though as again I don't have a better plan 🤷‍♂️

Reply:
I had to use this pattern for adding an extra action for a few rest grids in a client-app and ended up creating a separate list to hold these actions for setting `toolbarActions` and `menuActions` on the RestGridModel, which at least avoids the dupe.

I wonder if we should just put the add, edit, delete actions in some `defaultRestActions` constant or something so apps could do `toolbarActions: [...defaultRestActions, myAction]`, though if you didn't want the default ordering you'd still need to unroll the list. There is something to be said about having Add, Edit, Delete always together in that order whenever you see a RestGrid, with any non-default actions coming after, though I always tend to put Clone/Copy actions next to Edit when I am building something like this.

Could see us having a few pre-build lists of actions: `defaultRestActions` (Add, Edit, Delete), `allRestActions` (Add, Edit, Clone, Delete)

Reply:
Exactly - I was thinking that a `defaultRestActions` collection could be a small but meaningful aid to consistency - ala https://github.com/exhi/hoist-react/blob/395e285e9281373aad81574e1d6dca5c302fa24b/cmp/grid/GridModel.js#L90 in the grid context menu.  

But you are also right that it seems weird to have clone after delete....  

I vote let's leave it alone for now and go with the approach we have here (no changes to this code). We can feel it out a bit more - hesitant to add more utils or defaults if we're not really sure on it.  Thanks for discussing.